src/AzureAuth/CommandMain.cs

```diff
@@ -217,7 +217,7 @@ public Alias TokenFetcherOptions
             get { return this.authSettings; }
         }
 
-        private AuthMode CombinedAuthMode => this.AuthModes.Aggregate((a1, a2) => a1 | a2);
+        private AuthMode CombinedAuthMode => this.GetCombinedAuthMode();

```

Comment:
Rather than leave this method and have it call a new one, we can change the implementation of this method directly. Change the the `=>` to a `{` and pood, now you are have a normal method.